changeset.go

```diff
@@ -0,0 +1,34 @@
+package memdb
+
+// ChangeSet describes a set of changes to memDB tables performed during a
+// transaction.
+type ChangeSet []Mutation
```

Comment:
I understand `ChangeSet` and `Mutation`, but I don't understand why multiple Mutations are a ChangeSet. I think it would be more obvious if it would be the plural of one of the words.

```suggestion
type Changes []Change
```

This is what I like the most.